src/Illuminate/Database/Query/Grammars/SqlServerGrammar.php

```diff
@@ -249,7 +249,7 @@ public function supportsSavepoints()
      */
     public function getDateFormat()
     {
-        return 'Y-m-d H:i:s.000';
+        return 'Y-m-d H:i:s.u';
```

Comment:
@loren138 Have you tried just using the `+` sign here to ignore the microseconds? It should have the same effect as the existing hard-coded `000` so shouldn't break existing applications.

```php
return 'Y-m-d H:i:s+';
```